src/proof/mod.rs

```diff
@@ -110,16 +110,17 @@ pub trait Prover<'a, 'b, F: LurkField, C: Coprocessor<F>> {
         &self,
         multiframes: &'a [MultiFrame<'_, F, IO<F>, Witness<F>, C>],
     ) -> Result<SequentialCS<'a, F, IO<F>, Witness<F>, C>, SynthesisError> {
-        let res = multiframes
+        // Note: This loop terminates and throws an error on the first occurrence of `SynthesisError`.
```

Comment:
Throws has a specific meeting in Rust.
```suggestion
        // Note: This loop terminates and returns an error on the first occurrence of `SynthesisError`.
```